frontend/src/app/shared/components/tabs/scrollable-tabs/scrollable-tabs.component.html

```diff
@@ -1,4 +1,4 @@
-<div class="op-scrollable-tabs" [ngClass]="classes">
+<div class="op-scrollable-tabs -sticky" [ngClass]="classes">
```

Comment:
It's good if the parent sets the positioning of an element, that way a BEM block is independent of the place it is being used in. So I think it's better if you add this modifier to the parent context. Also, modifiers should be done with `&_sticky`, not `.-sticky`.